File: clients/Python/Example_17_1.py
import p5d

# Learning Processing
# Daniel Shiffman
# http://www.learningprocessing.com
#
# Example 17-1: Simple displaying text

def setup():
  global f
  pg.size(640, 480);

  # Step 3: Create Font
  f = pg.loadFont("fonts/lucida.txf")

def draw():
  global f
  pg.background(255)
  pg.textFont(f)     # Step 4: Specify font to be used
  pg.textSize(16)
  pg.fill(0)         # Step 5: Specify font color

  # Step 6: Display Text
  pg.text("Mmmmm ... Strings ...", 10, pg.height/2);

pg = p5d.PGraphics()
pg.setupFunc(setup)
pg.drawFunc(draw)
pg.listen()

File: clients/CPP/example17_1.cpp
#include "p5d.h"

// Learning Processing
// Daniel Shiffman
// http://www.learningprocessing.com

// Example 17-1: Simple displaying text

// Step 2: Declare PFont variable

//PFont f;

p5d::PGraphics pg("localhost", "8888");

void setup() {
  pg.size(480, 270);
  pg.frameRate(10);

  // Step 3: Create Font
  //f = createFont("Arial", 16);
}

void draw() {
  pg.background(255);
  pg.textSize(16); //textFont(f, 16); // Step 4: Specify font to be used
  pg.fill(0);         // Step 5: Specify font color

  // Step 6: Display Text
  pg.text("Mmmmm ... Strings ...", 10, pg.height/2);
}

int main(int argc, char **argv) {
  pg.setupFunc(setup);
  pg.drawFunc(draw);

  pg.listen();
}